<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(179, 179, 179)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="tabla_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>591</width>
      <height>461</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>591</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>1</y>
       <width>591</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>10</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Movimiento</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="caja_rfc_7">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit#caja_rfc_7{border: 1px solid rgb(0,0,0);};
</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cuenta</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cargo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Abono</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Referencia</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Concepto</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="caja_rfc_8">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#caja_rfc_8{border: 1px solid rgb(0,0,0);}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="caja_rfc_9">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#caja_rfc_9{border: 1px solid rgb(0,0,0);}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="caja_rfc_10">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#caja_rfc_10{border: 1px solid rgb(0,0,0);}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="caja_rfc_11">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#caja_rfc_11{border: 1px solid rgb(0,0,0);}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="caja_rfc_12">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>240</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#caja_rfc_12{border: 1px solid rgb(0,0,0);}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="caja_rfc_13">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>300</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#caja_rfc_13{border: 1px solid rgb(0,0,0);}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="boton_guardar">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>400</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(116, 170, 80);</string>
     </property>
     <property name="text">
      <string>Guardar</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="botones_frame">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QToolButton" name="minimizar">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-with 0px;</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../iconos/icons/minimize-window.png</normaloff>../iconos/icons/minimize-window.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="maximizar">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-with 0px;</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../iconos/icons/close-window.png</normaloff>../iconos/icons/close-window.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
